hypervisor/dm/vpci/vpci.c

```diff
@@ -721,10 +727,18 @@ int32_t vpci_assign_pcidev(struct acrn_vm *tgt_vm, struct acrn_assign_pcidev *pc
 			vdev_in_sos->vpci = vpci;
 
 			spinlock_obtain(&tgt_vm->vpci.lock);
-			vdev = vpci_init_vdev(vpci, vdev_in_sos->pci_dev_config, NULL);
+			vdev = vpci_init_vdev(vpci, vdev_in_sos->pci_dev_config, vdev_in_sos->phyfun);
 			pci_vdev_write_vcfg(vdev, PCIR_INTERRUPT_LINE, 1U, pcidev->intr_line);
 			pci_vdev_write_vcfg(vdev, PCIR_INTERRUPT_PIN, 1U, pcidev->intr_pin);
 			for (idx = 0U; idx < vdev->nr_bars; idx++) {
+				/* VF physical BARs are zero & RO, initialze assigned VF BARs from SOS */
```

Comment:
This comment seems to be irrelevant, is it? Delete this line of comment?

Reply:
OK, I will remove it